pyproject.toml

```diff
@@ -58,12 +58,72 @@ Documentation = "https://pymediainfo.readthedocs.io/"
 Bugs = "https://github.com/sbraz/pymediainfo/issues"
 
 
+# Custom entries for downloading the bundled libmediainfo
+[tool.bundled_libmediainfo]
+version = "24.12"
+
+[[tool.bundled_libmediainfo.wheel]]
+platform = "linux"
+arch = "x86_64"
+blake2b_sums = "13c4afb2948187cc06f13b1cd7d7a49f8618b8d1e3a440d9e96ef7b486a653d5e2567aae97dc253e3d3f484a7837e4b5a972abab4803223300a79c601c0bcce1"
+tag = "manylinux_2_27_x86_64"
+
+[[tool.bundled_libmediainfo.wheel]]
+platform = "linux"
+arch = "arm64"
+blake2b_sums = "4e5a9826fa987f4bde46a6586894d45f3d5381f25d8886dfef67f5db3a9f4377ecafc12acc6e2d71e43b062b686c2db2523052a1f3dd7505a41091847788d114"
+tag = "manylinux_2_27_aarch64"
+
+[[tool.bundled_libmediainfo.wheel]]
+platform = "darwin"
+# The same file is used for darwin x86_64 and arm64 (suffixed Mac_x86_64+arm64.tar.bz2)
+arch = "arm64"
+blake2b_sums = "65b8195f0859369fa0ab1870cbde1535bb57f16bde451b22585d849c870f1ca92972328c11edd15b6f8187445dd58efff7107cfce39d2be7a88e8c434589b4ae"
+tag = "macosx_10_10_universal2"
+
+[[tool.bundled_libmediainfo.wheel]]
+platform = "win32"
+arch = "x86_64"
+blake2b_sums = "f831c588e9eaf51201b4cc7995dce66852591764fc5ef05effd3a8a2037ff5d37ec039eef5d1f990f05bd7452c1cad720e95b77a095f9f1a690689e351fc00b8"
+tag ="win32"
+
+[[tool.bundled_libmediainfo.wheel]]
+platform = "win32"
+arch = "i386"
+blake2b_sums = "0f0e14c103eac858fe683ec7d51634d62e5e0af658940fd26608249f1048846a92a334438204fe5ecfceb70cb00e5550bfb717a77f10816a2583b5041bb61790"
+tag = "win_amd64"
+
+[[tool.bundled_libmediainfo.wheel]]
+platform = "win32"
+arch = "arm64"
+blake2b_sums = "e9f426f2e873f65ca1fdede5f8eb6dc3e871e682b821ad3f62acb1428eda5a27ce363f77a2c621d1b15f8e9b94976a10afb56fc069325ca7ba4efad14df03105"
+tag = "win_arm64"
```

Comment:
`libmediainfo` is compiled for windows arm64, it doesn't cost much to bundle a wheel for this architecture.
Well, it has to be added to `appveyor.yaml`